avalon/pipeline.py

```diff
@@ -1,407 +1,17 @@
 """Core pipeline functionality"""
 
-import os
 import sys
-import json
-import types
 import logging
-import inspect
-import importlib
 
-from . import (
-    io,
-
-    Session,
-
-    _registered_host,
-    _registered_root,
-    _registered_config,
-)
 
 self = sys.modules[__name__]
-self._is_installed = False
-self._config = None
 self.data = {}
 # The currently registered plugins from the last `discover` call.
 
 log = logging.getLogger(__name__)
```

Comment:
These remaining lines in `pipeline.py` can also already be removed, no?

I actually don't think that `publish()` line is used anywhere either from `avalon.pipeline.publish` or `avalon.api.publish`. The quick code search I did seems to find only direct usages of `pyblish.util.publish()`.
Which means that technically with this PR we could just as well remove the full `pipeline.py` file?

Reply:
Good point, that's true.